library/cartridge_validate_config.py

```diff
@@ -467,9 +466,9 @@ def validate_config(params):
             return ModuleRes(failed=True, msg=errmsg)
 
         if host_vars.get('stateboard') is True:
-            if found_stateboard_vars is not None:
-                return ModuleRes(failed=True, msg='Only one instance can be marked as a "stateboard"')
```

Comment:
Now it's possible to use several stateboards?

Reply:
It was always possible to start one stateboard on each machine. Anyway, you can specify only one stateboard connect options in `failover_params`. But starting more than one stateboard allows me to test that stateboard starts correctly for TGZ package (multiversion and not) as well as for RPM and DEB.